BossMod/Components/LineOfSightAOE.cs

```diff
@@ -110,6 +110,7 @@ public override void OnCastStarted(BossModule module, Actor caster, ActorCastInf
             {
                 _casters.Add(caster);
                 Refresh(module);
+                NextExplosion = spell.NPCFinishAt;
```

Comment:
This isn't right - this should be done by Refresh(), using active caster's NPCFinishAt.

Reply:
I guess I will remove it for now. I don't really understand why this would be needed since it seems to work properly when looking at AI hints in replay.